src/components/ProductBlock/ProductDesc/ProductTabs/Tabs/index.tsx

```diff
@@ -0,0 +1,79 @@
+import classNames from 'classnames';
+import React, { useState, useEffect, useRef } from 'react';
+
+import { ITab } from '@utils/tab';
+
+import './Tabs.scss';
+
+type PropTypes = {
+  tabs: ITab[];
+  selectedId: string | number;
+  onClick?: (id: string | number) => void;
+  className?: string;
+};
+
+export const Tabs: React.FC<PropTypes> = ({
+  className,
+  onClick,
+  selectedId,
+  tabs,
+}) => {
+  const [animateStyle, setAnimateStyle] = useState({});
+
+  const tabRef = useRef<HTMLDivElement>(null);
+
+  useEffect(() => {
+    const selectedTab = tabRef.current?.querySelector(
+      '.tabs__tab--selected',
+    ) as HTMLElement;
+
+    const recalculatePosition = () => {
+      if (selectedTab) {
+        const { width, left } = selectedTab.getBoundingClientRect();
+        const containerLeft = tabRef.current?.getBoundingClientRect().left || 0;
+
+        setAnimateStyle({
+          width,
+          transform: `translateX(${left - containerLeft}px)`,
+        });
+      }
+    };
+
+    recalculatePosition();
+
+    const handleResize = () => {
+      recalculatePosition();
+    };
```

Comment:
?

Reply:
i've wrote in this way that the strip under tabs worked correctly
but I'll rewrite it